### PR TITLE
Card reviews update

### PR DESCRIPTION
**Pull request by poojakelkar**
badge

- Added card with badge
- card with dismiss

### CODE REVIEW

File: components/button/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
```
Comment (sahiba0915) [inline components/button/index.html]: give proper title.

File: components/image/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
```
Comment (sahiba0915) [inline components/image/index.html]: give proper title

File: components/input/index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (sahiba0915) [inline components/input/index.html]: give proper title

**Review by sahiba0915 — APPROVED** [review]
Added few suggestions, rest code looks clean and good class names used, rest good to go

File: components/button/index.html (hunk 1 above)
Comment (poojakelkar) [reply components/button/index.html]: yeah, Its changed now thank you

File: styles.css [hunk 4]
```diff
@@ -357,7 +357,7 @@ body {
     color: var(--nav-color);
     border: none;
     border-radius: 5px;
```
Comment (chanchal-panpaliya) [inline styles.css]: For border radius, see https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius. parent is "rem" unit, then the border radius should be in rem only.

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good . read doc for border radius .

File: styles.css (hunk 4 above)
Comment (poojakelkar) [reply styles.css]: yeah Thanks

**Review by aliamaanansari — APPROVED** [review]
Looks good to me

**Pull request merged**